README.md

```diff
@@ -54,6 +53,23 @@ phoila install ibis-vega-transform
 phoila "examples/Charting Example.ipynb"
 ```
 
+### Tracing
+
+If you want to see traces of the interactiosn for debugging and performance analysis,
+install tthe `jaeger-all-in-one` binary and the `jupyterlab-server-proxy-saulshanabrook`
+lab extension to see the Jaeger icon in the launcher.
```

Comment:
Should we install this by default on the `binder/environment.yml`?

Also, now it should read
```
# For mac and linux
`conda install jaeger -c conda-forge`
```

Reply:
Yep and yep! Just opened an issue for it https://github.com/Quansight/ibis-vega-transform/issues/31